### PR TITLE
Add possibility for additional email fields in email lists

### PR DESCRIPTION
**Pull request by erlingfn**
Make it possible to add emails for people outside of abakus to email lists manually

### CODE REVIEW

**Comment by mrfarstad** [comment]
Nice work! Is the dependency upgrade related to this PR? Maybe move the upgrading to another PR. That way it would be easier to revert the PR if the upgrading causes issues. 😁

**Comment by erlingfn** [comment]
Thanks for the feedback, I believe I have made the changes you requested now!

File: app/routes/admin/email/components/EmailListEditor.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 import React from 'react';
 import Button from 'app/components/Button';
-import { createValidator, required } from 'app/utils/validation';
+import { createValidator, required, EMAIL_REGEX } from 'app/utils/validation';
```
Comment (JonasBak) [inline app/routes/admin/email/components/EmailListEditor.js]: There is a `isEmail` function here you can import instead of the "raw" regex

Comment (erlingfn) [reply app/routes/admin/email/components/EmailListEditor.js]: There is, however I could not find a way to use it to validate an array of emails

Comment (odinuge) [reply app/routes/admin/email/components/EmailListEditor.js]: :+1: Should make some logic for validating arrayfields

Comment (odinuge) [reply app/routes/admin/email/components/EmailListEditor.js]: Out of scope here tho.

**Pull request merged**